### PR TITLE
Docs/SK-691 | Add tutorial for distributed deployment

### PR DESCRIPTION
**Pull request by viktorvaladi**
Adds a tutorial to the docs for distributed deployment.

### CODE REVIEW

**Review by ahellander — CHANGES_REQUESTED** [review]
Nice work. However, I was more envisaging a more detailed step-by-step tutorial showing how to configure a RPI4 with FEDn and starting it as a client, probably connecting it to an existing FEDn deploy and FEDn Studio.

**Comment by ahellander** [comment]
So I think it will be more pedagogical to split in two: 

1. Distributed deployment (docs) 
2. Example / tutorial with Rasberry PI5.

**Comment by viktorvaladi** [comment]
> So I think it will be more pedagogical to split in two:
> 
> 1. Distributed deployment (docs)
> 2. Example / tutorial with Rasberry PI5.

Sounds good! I will fix that.

File: docs/distributed.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+-  `One host workstation and atleast one client device`
+-  `Python 3.8, 3.9 or 3.10 <https://www.python.org/downloads>`__
+-  `Docker <https://docs.docker.com/get-docker>`__
+-  `Docker Compose <https://docs.docker.com/compose/install>`__
+
+Launch a distributed FEDn Network 
+-------------
+
+
+Note the local IP address of your host device; it will be needed later. Follow the standard procedure
```
Comment (ahellander) [inline docs/distributed.rst:28]: Better here to replicate how you start up the server-side? Suggest the user to use docker-compose? Also mention where to find the ports they need to open?

File: docs/distributed.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+   —add-host=api-server:<host local ip> \
+   —add-host=combiner:<host local ip> \
+   <image name> run client -in client.yaml --name client1
+
+
+Alternatively updating the `/etc/hosts` file, appending the following lines for running naitively:
+
+.. code-block::
+
+   <host local ip>      api-server
```
Comment (ahellander) [inline docs/distributed.rst:60]: Should be public ip, right?

**Review by ahellander — COMMENTED** [review]
Also, I suggest that you highlight that this is not a secure deployment (unless the users takes separate measures) and that you refer to FEDn Studio for a production-grade deployment.

File: docs/distributed.rst (hunk 2 above)
Comment (viktorvaladi) [reply docs/distributed.rst:63]: This is mainly for deployment over local networks, so the host's local network IP. I've clarified that and referred to Studio for secure deployment.

File: docs/distributed.rst (hunk 1 above)
Comment (viktorvaladi) [reply docs/distributed.rst]: I suggested docker-compose and added where to find the ports in the note.

File: docs/distributed.rst (hunk 2 above)
Comment (ahellander) [reply docs/distributed.rst:63]: If it is a distributed setup you cannot communicate on the hosts local IP from a client. I think the setting that all machines are on the same local network is fairly uncommon.

Comment (ahellander) [reply docs/distributed.rst:63]: Ok, I see, you made that clear in the beginning of the tutorial.

**Review by ahellander — APPROVED** [review]

**Pull request merged**